hubspot/connection/__init__.py

```diff
@@ -228,11 +230,11 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._session.close()
 
 
-_AuthenticationKey = Record.create_type('_AuthenticationKey', 'key_value')
+_AuthenticationKey = Record.create_type(str('_AuthenticationKey'), str('key_value'))
```

Comment:
Sounds good.  I'll remove these.  I agree that it is a little baffling and that the best place to deal with it is in pyrecord.